sherpa/utils/tests/test_utils_unit.py

```diff
@@ -566,3 +569,89 @@ def test_bool_cast_invalid(arg, badval):
     emsg = arg if badval is None else badval
     with pytest.raises(TypeError, match=f"^unknown boolean value: '{emsg}'$"):
         bool_cast(arg)
+
+
+@pytest.mark.parametrize("x,mu,sigma,eclass,emsg",
+                         [([1, 2], 3, [3], TypeError, "x and mu sizes do not match"),
+                          ([3], [4, 5], [3], TypeError, "x and mu sizes do not match"),
+                          # This one is not explicitly checked for, so the
+                          # error message may change with NumPy; so we may
+                          # decide not to test it if it needs changing
+                          ([2, 3], [3, 4], [2, 3],
+                           ValueError, "diag requires an array of at least two dimensions"),
+                          ([2, 3, 4], [3, 4, 6], [[2, 3], [3, 4]],
+                           TypeError, "sigma shape does not match x"),
+                          ([2, 3], [3, 4], [[1, 2], [3, 4]],
+                           ValueError, "sigma is not positive definite"),
+                          ([2, 3], [3, 4], [[1, -2j], [2j, 5]],
+                           ValueError, "sigma is not symmetric")
+                          ])
+def test_multinormal_pdf_invalid_input(x, mu, sigma, eclass, emsg):
+    """Minimal test of invalid input"""
+
+    with pytest.raises(eclass, match=emsg):
+        multinormal_pdf(x, mu, sigma)
+
+
+# The values were calculated using CIAO 4.16 on a Linux x86_64 OS.
+#
+@pytest.mark.parametrize("x,mu,sigma,expected",
+                         [([2, 1], [0.5, 0.7], [[1, 0], [0, 1]],
+                           0.049396432874713896),
+                          ([1, 2], [0.5, 0.7], [[1, 0], [0, 1]],
+                           0.06033293935644923)])
+def test_multinormal_pdf(x, mu, sigma, expected):
+    """Very basic regression test"""
+
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore",
+                                message="the matrix subclass",
+                                category=PendingDeprecationWarning)
+        out = multinormal_pdf(x, mu, sigma)
```

Comment:
You removed the matrix use here, so this filter should not be necessary.

Reply:
well, only possible >= 53f678611916fe2961fc766802bb6d8925100e9a ....

Reply:
I've removed the checks. Ta for the review.